system-addon/test/unit/lib/FilterAdult.test.js

```diff
@@ -14,7 +15,6 @@ describe("filterAdult", () => {
         return hashStub;
       }
     };
-    filterAdult = require("lib/FilterAdult.jsm").filterAdult;
```

Comment:
Is there a reason this is re-imported/re-assigned once per test @Mardak?

Reply:
Not that I can remember. A quick glance maybe points to the lazy `gCryptoHash = Cc["@mozilla.org/security/hash;1"].createInstance(Ci.nsICryptoHash);` ? I believe at one point it used some `XPCOM.defineLazy…`something, but I suppose if the tests pass, then no problem!